src/sentry/static/sentry/app/components/sidebar/index.jsx

```diff
@@ -78,6 +79,15 @@ class Sidebar extends React.Component {
 
     this.hashChangeHandler();
     this.doCollapse(this.props.collapsed);
+
+    const {organization} = this.props;
+    logExperiment({
+      organization,
+      key: 'OnboardingSidebarV2Experiment',
+      unitName: 'org_id',
+      unitId: parseInt(organization.id, 10),
```

Comment:
do you still need the `unitId`?

Reply:
Not anymore, but I would like to fix these all at once.

Reply:
I'm killing unitName aswell

Reply:
hm I kind of like having `unitName`. how are you differentiating between user and org experiment logging?

Reply:
https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/data/experimentConfig.tsx they are marked here